eagleEvents/templates/seating-chart.html.j2

```diff
@@ -0,0 +1,211 @@
+{% extends "layout.html.j2" %}
+{% block head_content %}
+      <title>Eagle Events Seating Chart</title>
+{% endblock %}  
+{% block body_content %}
+    <nav class="navbar navbar-default">
+        <div class="container-fluid">
```

Comment:
I don't think this page should have a nav bar, that should be taken care of by the layout page

Reply:
Removed the Navbar from the page.